ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -183,12 +177,21 @@ export OPREFIX=gdas.t${cyc}z
 export APREFIX=gdas.t${cyc}z
 export GPREFIX=gdas.t${gcyc}z
 
+if [ $obtype = "conv_ps" ]; then
+cat > $workdir/temp.yaml << EOF
+window begin: '2021-07-31T00:00:00Z'
+window end: '2021-08-01T21:00:00Z'
+observations:
+- !INC $yamlpath
+EOF
+else
```

Comment:
@CoryMartin-NOAA This is a temporary fix.  Please advise if you have any suggeston.  Thanks.

Reply:
@emilyhcliu is this making the window larger because of the convention difference between JEDI and GSI? 
If so,
try:
```
window shift: True
```
after line 190

Reply:
@CoryMartin-NOAA Yes.  The hack is used fue to convention difference between JEDI and GSI.
Your suggestion worked.  Thanks a lot!!
I will modify it accordingly.